src/client/contextChannels.ts

```diff
@@ -158,8 +174,8 @@ export interface ChannelContextListener extends ContextListener {
 /**
  * Class representing a context channel. All interactions with a context channel happen through the methods on here.
  *
- * When users wish to generically handle both {@link SystemChannel}s and the {@link DefaultChannel}, generally the
- * {@link Channel} type should be used instead of {@link ChannelBase}.
+ * When users wish to generically handle both {@link SystemChannel}s, {@link AppChannel}s and the
+ * {@link DefaultChannel}, generally the {@link Channel} type should be used instead of {@link ChannelBase}.
```

Comment:
Below, change "reset back into it's initial context-less state" to "reset back into *its* initial context-less state"